engine/src/cmd/base_interface.cpp

```diff
@@ -391,17 +390,16 @@ void BaseInterface::Room::Draw(BaseInterface *base) {
     const bool enable_markers = configuration()->graphics.bases.enable_location_markers;
     const bool draw_text = configuration()->graphics.bases.draw_location_text;
     const bool draw_always = configuration()->graphics.bases.location_marker_draw_always;
-    static float y_lower =
-            -0.9;           //shows the offset on the lower edge of the screen (for the textline there) -> TODO: Should be defined globally somewhere
-    const float base_text_background_alpha = configuration()->graphics.bases.text_background_alpha;
+    static float y_lower = kYLower;           //shows the offset on the lower edge of the screen (for the textline there)
```

Comment:
Why have it here if you can just access kYLower from below?

Reply:
Good point